cmd/thv-operator/api/v1alpha1/virtualmcpserver_types.go

```diff
@@ -328,6 +328,15 @@ type ErrorHandling struct {
 
 // OperationalConfig defines operational settings
 type OperationalConfig struct {
+	// LogLevel sets the logging level for the Virtual MCP server.
+	// Supported values: info, debug, warn, error (default: info)
+	// Note: warn and error levels are accepted but currently behave like info
+	// until vmcp binary adds support for these levels.
+	// +kubebuilder:validation:Enum=info;debug;warn;error
+	// +kubebuilder:default=info
+	// +optional
+	LogLevel string `json:"logLevel,omitempty"`
```

Comment:
Do you think we need to have the three-choice LogLevel if we only ever use the debug value? Shouldn't we just use Debug bool?